### ISSUE TITLE
model Profiles is deleted following Initial Setup

### ISSUE THREAD

**Issue by devallibus** [Jaaneek/t3-supabase-app-router#15]
Hello,

The Profiles model located in Prisma's schema is getting deleted while following the README's Initial Setup.

Seeing the comments, I have attempted to run the stored Supabase migration on a new Supabase project, attempting to then be able to follow the Initial Setup or even just db push without nuking the Profiles model, but with no success.

Either I'm being obtuse, or I suspect I won't be the only one getting blocked by this?

Any help would be greatly appreciated, thank you.

**Comment by Jaaneek** [comment]
Hi,

Are you running pnpm prepare:local?

**Comment by devallibus** [comment]
> Hi,
> 
> Are you running pnpm prepare:local?

I am only using prepare:remote, as I don't develop with local supabase.

Starting from scratch, and with a new Supabase project, as soon as I run prepare:remote, it will modify the schema.

After that I tried running the migration stored with `supabase db push` and `db:push` before. Required several tries but seems I made it work finally.

The schema is still modified by prisma, but only adding several `@map `

I am on mobile right now so I can't give more details, but happy to once I'm on desktop.

Thank you.
